ios/Classes/SwiftChargebeeFlutterSdkPlugin.swift

```diff
@@ -62,7 +62,9 @@ public class SwiftChargebeeFlutterSdkPlugin: NSObject, FlutterPlugin {
                         CBPurchase.shared.purchaseProduct(product: product, customerId: customerId) { result in
                             switch result {
                             case .success(let result):
-                                let dict = ["status": "\(result.status)", "subscriptionId": "\(result.subscriptionId)", "planId": "\(result.planId)"]
+                                if let subscriptionId = result.subscriptionId, let planId = result.planId{
+                                    dict = ["status": "\(result.status)", "subscriptionId": "\(subscriptionId)", "planId": "\(planId)"]
+                                }
```

Comment:
Should we handle the case when subscriptionId/plandId is not present?

Reply:
We should always get planId & subscriptionId once purchase done. Added if let not to pass Optional value to flutter.